django/http/request.py

```diff
@@ -61,6 +61,8 @@ def __init__(self):
         self.method = None
         self.resolver_match = None
         self._post_parse_error = False
+        self.content_type = None
+        self.content_params = None
```

Comment:
I'd prefer None, especially for the params, as it clearly indicates we did not parse a value, as opposed to "parsed an empty value"... which is more important for the params than the header [where I assume it would be an invalid request]

All the other edits polishing my docs seem sane.